webpack.config.ts

```diff
@@ -27,8 +32,18 @@ const base = async (env?: Partial<WebpackEnvParams>): Promise<Configuration> =>
       'process.env.ENV': env?.mode || 'development',
       'process.env.IS_WEBPACK_BUILD': 'true',
     }),
+    new NodeProtocolUrlPlugin(),
+    new ProvidePlugin({
+      path: stdLibBrowser.path,
+      fs: stdLibBrowser.fs,
+      stream: stdLibBrowser.stream,
+      crypto: stdLibBrowser.crypto,
+      process: stdLibBrowser.process,
+      Buffer: [stdLibBrowser.buffer, 'Buffer']
```

Comment:
we just had an agreement not long ago the polyfills should be done by the application that uses fdp-storage.